tests/unit/azure/test_resources.py

```diff
@@ -13,29 +13,31 @@
     StorageAccount,
 )
 
-from . import azure_api_client, get_az_api_mapping
+from . import azure_api_client as create_azure_api_client, get_az_api_mapping
```

Comment:
I needed to update the API client in the `test_access.py`, was on the way to make it a fixture but decided to leave the reminder for https://github.com/databrickslabs/ucx/issues/2949